fem/eltrans.hpp

```diff
@@ -341,12 +349,23 @@ class IntegrationPointTransformation
    void Transform (const IntegrationRule  &, IntegrationRule  &);
 };
 
-class FaceElementTransformations
+class FaceElementTransformations : public IsoparametricTransformation
 {
+private:
+   int side;
+
 public:
-   int Elem1No, Elem2No, FaceGeom;
-   ElementTransformation *Elem1, *Elem2, *Face;
+   int Elem1No, Elem2No;
+   ElementTransformation *Elem1, *Elem2;
    IntegrationPointTransformation Loc1, Loc2;
+
+   FaceElementTransformations() : side(2) {}
+
+   int SetActiveSide(int s);
```

Comment:
This active active side concept should be explained with some doxygen.

Reply:
I agree.  This "side" concept was introduced in the relevant `GridFunction::GetFaceValue` methods but it isn't documented anywhere.  A doxygen block in the `FaceElementTransformations` class is a good place to do this.  I'll take care of it.